packages/back-end/src/jobs/sdkWebhooks.ts

```diff
@@ -161,29 +161,42 @@ async function runWebhookFetch({
   const signingKey = webhook.signingKey;
   const headers = webhook.headers || "";
   const method = webhook.httpMethod || "POST";
-  const sendPayload = webhook.sendPayload;
+  // todo: better inference based on sendPayload
+  const payloadFormat = webhook.payloadFormat || "standard";
   const organizationId = webhook.organization;
   const requestTimeout = 30000;
   const maxContentSize = 1000;
 
+  const sendPayload =
+    method !== "GET" && ["standard", "sdkPayload"].includes(payloadFormat);
```

Comment:
This should ensure that the keys in the array are actually payload formats, in case they change in the future etc.
```suggestion
    method !== "GET" && (["standard", "sdkPayload"] as const).includes(payloadFormat);
```

Reply:
you could also make this a const above like it makes it easier to read compaired to the inline const 
```
const canSendPayloadFormats =  ["standard", "sdkPayload"];
const sendPayload =
    method !== "GET" && canSendPayloadFormats.includes(payloadFormat);
```